doc/manuals/taskvine/index.md

```diff
@@ -225,6 +225,13 @@ Here is how to describe the files needed by this task:
     # when the name at manager is the same as the exection site, we can write instead:
     t.add_input_file("my-file",     cache = False)
     t.add_output_file("my-file.gz", cache = False)
+
+    # files can also be described when the task is declared:
+     t = vine.Task(
+        command = "./gzip < my-file > my-file.gz",
+        input_files = {"/usr/bin/gzip" : {remote_name : "gzip, cache : True}, "my-file" : {remote_name : "my-file", cache : False}},
```

Comment:
Given this is a doc example, I suggest having one key per line, as:
```python
input_files = {
    gzip_file: {
        "remote_name": "..."
   ...
}

Reply:
Fixed